src/gui/static/src/app/components/pages/settings/backup/backup.component.html

```diff
@@ -2,25 +2,30 @@
   <app-header title="Backup"></app-header>
 
   <mat-card>
-    <p>Wallet Directory: {{ folder }}</p>
+    <p>
+      Wallet Directory: {{ folder }}
+      <mat-icon matTooltip="Wallet Directory contains .wlt files with seeds.
+      You will be able to read only unencrypted wallet files.
+      Use the table below to get seeds from your encrypted wallets.">help_outline</mat-icon>
+    </p>
 
     <p>BACKUP YOUR SEED. ON PAPER. IN A SAFE PLACE. As long as you have your seed, you can recover your coins.</p>
   </mat-card>
 
   <ngx-datatable #table
                  class="material"
-                 [rows]="walletService.all() | async"
+                 [rows]="onlyEncrypted"
```

Comment:
You can find seed from unencrypted wallets in their .wlt files. To see the seed of encrypted wallet, you have to do it from the app, since it has to decrypt the file first.